src/weather_module.js

```diff
@@ -0,0 +1,37 @@
+import {
+  celciusDisplay,
+  fahrenheitDisplay,
+  displayError,
+  changeWeatherImg,
+  flashInfo,
+} from './dom_module';
+
+const WeatherModule = (function theWeatherModuleFunction() {
+  const mod = {};
+  mod.getWeatherInfos = function collectWeatherFromLocation(location, units = 'metric') {
+    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=617ec88a7d93c5b6ace5a90f054e9838&units=${units}`, { mode: 'cors' })
+      .then((response) => response.json())
```

Comment:
That is a strange stickler error. Anyways according to the code style guide we should remove the parentheses.